actions/chat/postMessage.ts

```diff
@@ -54,8 +62,42 @@ export async function postChatMessage(
     plugin?.voterWeightPk
   )
 
-  const transaction = new Transaction()
-  transaction.add(...instructions)
+  // the list will have element only if the plugin is NFTVoterClient2
+  // so we can just add the chuncks to the instructionsChunks
+  const nftTicketAccountsChuncks = chunks(createPostMessageTicketIxs, 1)
+  const postMessageIxsChunk = [instructions]
+
+  const instructionsChunks = [
+    ...nftTicketAccountsChuncks.map((txBatch, batchIdx) => {
+      return {
+        instructionsSet: txBatchesToInstructionSetWithSigners(
+          txBatch,
+          [],
+          batchIdx
+        ),
+        sequenceType: SequenceType.Parallel,
```

Comment:
hmm i think the way you are doing this will make it such that the post message stuff might happen too soon relative to the ticket creation, due to sequencetype parallel here

Reply:
Isn't the postMessageIxsChunk will happen after all the parallel txs are done?

Reply:
the short answer is that i dont know and i find the mangolana source code too hard to follow. you should ask Adrian